examples/common/idxio.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
```

Comment:
https://stackoverflow.com/questions/5709616/whats-the-difference-between-these-two-python-shebangs

Reply:
I think the env variant might be better